source/common/network/lc_trie.h

```diff
@@ -309,7 +311,7 @@ template <class T> class LcTrie {
    */
   template <class IpType, uint32_t address_size = CHAR_BIT * sizeof(IpType)> class BinaryTrie {
   public:
-    BinaryTrie() : root_(std::make_unique<Node>()) {}
+    BinaryTrie(bool exclusive) : root_(std::make_unique<Node>()), exclusive_(exclusive) {}
 
     /**
      * Add a CIDR prefix and associated tag to the binary trie. If an entry already
```

Comment:
[minor] You might want to `s/tag/data/g` in this comment for consistency with the new nomenclature.